### ISSUE TITLE
Item tag filter

### ISSUE THREAD

**Issue by bmstasa** [LittleLightForDestiny/littlelight#91]
Hello,

Enjoying the app.  Very helpful.  I tagged several items as "trash" to delete in game.  Was going to filter by tag, move to character and delete, but the filter by tag is greyed out.  Is this a bug or intentional?

Thanks!

**Comment by OKBobby** [comment]
I have the same problem.  I noticed it when tagging Kinetic Weapons.

**Comment by Sleazus** [comment]
If you haven't figured it out nearly 2 years later, you need 2 different tags to filter.
They should let you filter if only one tag is selected but no.

Just tag a weapon you're currently using as a favourite and you'll be able to select between favourite and trash

**Comment by joaopmarquesini** [comment]
wait what ? @Sleazus is this still happening ?

**Comment by Sleazus** [comment]
Yeah, I tagged something as trash and nothing else and the filtering was greyed out.
Searched the web found this, then decided to see if it would let me sort of something else was favourited. It worked

**Comment by bmstasa** [comment]
I recall this issue getting fixed, however I stopped using the app about a year ago as there was an issue syncing information from my account.  The app would get stuck on a continuous loading screen.  Uninstall / reinstall didn't fix and Joao wasn't able to troubleshoot on his end.  Surprisingly I just tried the app again and it works!  Awesome!  Something got resolved behind the scenes.  Thank you, Joao!  I also tried tagging and filtering a few combination of weapons, it appears to work how I expected previously.
